File: lessons/compiled_lang/java/src/j10.arguments.java
// Not public - see a00.output.java for why.
class J10Arguments {
    public static void main(String[] args) {
        System.out.println("The arguments passed are:");
        for (int i = 0; i < args.length; i++) {
            System.out.println(" item " + (i + 1) + ": " + args[i]);
        }
    }
}

File: lessons/compiled_lang/cs/src/j10.arguments.cs
using System;

class J10Arguments
{
    static void Main(string[] args)
    {
        Console.WriteLine("The arguments passed are:");
        for (int i = 0; i < args.Length; i++)
        {
            Console.WriteLine(" item " + (i + 1) + ": " + args[i]);
        }
    }
}
